python/desc/sims/GCRCatSimInterface/CatalogClasses.py

```diff
@@ -28,7 +28,7 @@ def get_sedFilepath(self):
     def get_shorterFileNames(self):
         """
         Method to truncate filenames for transient
-        spectra written out by phosim. 
+        spectra written out by phosim.
```

Comment:
Didn't you change these whitespace errors on another PR I approved this morning?

....I guess we'll find out when we merge.